### DISCUSSION TITLE
[M-Z] (13 Giugno 2022) Esercizio 8

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#472] [Esami & Esoneri M-Z]
### Traccia

<img width="907" alt="image" src="https://github.com/user-attachments/assets/87bd0a51-400a-416c-ad96-ccaea3b183bf">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20220613.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20220613.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/d1279765-2434-4137-9fe8-a6f27bee76a9)

## Tableau

![image](https://github.com/user-attachments/assets/8a1b543c-c3d5-4071-9381-d73d9ffa500e)
